CHANGELOG-zh-CN.md

```diff
@@ -1,5 +1,147 @@
 # OceanBase Developer Center (ODC) CHANGELOG
 
+## 4.2.3 (2023-12-15)
+
+### 功能变化
+
+导入导出
+
+- 支持原生 MySQL 数据源的导入导出
+- OceanBase 数据源的导入导出任务配置页面中不再提供 SYS 账户配置
+
+数据库对象管理
+
+- OceanBase MySQL 以及原生 MySQL 模式下支持 GIS 数据类型
+- 白屏创建或删除索引时给出高风险操作提示
+
+数据源
```

Comment:
1. move datasource category to the top.

2. ODC 4.2.3 new features may miss:
- SQL Diagnose: a lot of improvement for Full Chain TRACE.
- SQL Execution:  snippet support 100k maxium size,  support ctrl+enter shortcut for execute current statement.
- System ingegration: bastion integration can cowork with SQL Check  integraiton now.
- DLM: support change parameter, support view task log .
- Datasource: support clone operation, for fast create a new datasource

Reply:
done